src/transformers/quantizers/quantizer_bnb_8bit.py

```diff
@@ -64,17 +65,26 @@ def __init__(self, quantization_config, **kwargs):
             self.modules_to_not_convert = self.quantization_config.llm_int8_skip_modules
 
     def validate_environment(self, *args, **kwargs):
-        if not torch.cuda.is_available():
-            raise RuntimeError("No GPU found. A GPU is needed for quantization.")
-
-        if not is_accelerate_available():
+        if not (is_accelerate_available() and is_bitsandbytes_available()):
```

Comment:
Not sure why this was reverted
```suggestion
        if not is_accelerate_available():
```